e2e-tests/package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "e2e-tests",
+  "version": "0.0.0",
+  "description": "End to end tests for boundary UI",
+  "author": "HashiCorp",
+  "license": "BUSL-1.1",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
+    "lint:js": "eslint .",
+    "lint:js:fix": "eslint . --fix",
+    "format": "concurrently \"npm:format:*\" --names \"format:\"",
+    "format:js": "prettier --write './**/*.{js,mjs}'",
+    "admin": "yarn playwright test --config admin/playwright.config.js",
+    "admin:ce:aws": "yarn run e2e:admin --grep \"(?=.*@ce)(?=.*@aws)\"",
+    "admin:ce:docker": "yarn run e2e:admin --grep \"(?=.*@ce)(?=.*@docker)\"",
+    "admin:ent:aws": "yarn run e2e:admin --grep \"(?=.*@ent)(?=.*@aws)\"",
+    "admin:ent:docker": "yarn run e2e:admin --grep \"(?=.*@ent)(?=.*@docker)\"",
+    "admin:dev:chrome": "yarn run e2e:admin --project=chromium --headed",
+    "admin:dev:firefox": "yarn run e2e:admin --project=firefox --headed",
+    "admin:dev:safari": "yarn run e2e:admin --project=webkit --headed",
```

Need to remove the `e2e` from those subcommands
```suggestion
    "admin:ce:aws": "yarn run admin --grep \"(?=.*@ce)(?=.*@aws)\"",
    "admin:ce:docker": "yarn run admin --grep \"(?=.*@ce)(?=.*@docker)\"",
    "admin:ent:aws": "yarn run admin --grep \"(?=.*@ent)(?=.*@aws)\"",
    "admin:ent:docker": "yarn run admin --grep \"(?=.*@ent)(?=.*@docker)\"",
    "admin:dev:chrome": "yarn run admin --project=chromium --headed",
    "admin:dev:firefox": "yarn run admin --project=firefox --headed",
    "admin:dev:safari": "yarn run admin --project=webkit --headed",
```